src/command/get_version.rs

```diff
@@ -0,0 +1,52 @@
+use anyhow::bail;
+use anyhow::Result;
+use log::warn;
+use semver::Version;
+
+use crate::git::error::TagError;
+use crate::CocoGitto;
+
+impl CocoGitto {
+    pub fn get_latest_version(
+        &self,
+        fallback: Option<String>,
+        fail_on_fallback: bool,
+        package: Option<String>,
+    ) -> Result<()> {
+        let fallback = match fallback {
+            Some(input) => match Version::parse(input.as_ref()) {
+                Ok(version) => version,
+                Err(err) => {
+                    warn!("Invalid fallback: {}", input);
+                    bail!("{}", err)
+                }
+            },
+            None => Version::new(0, 0, 0),
```

Comment:
I'm not sure what the purpose of this default fallback value is. Could you make a short example where it's useful?

Reply:
In case of a fresh repository (only `cog init` has been run), one may want to fall back to a default version. 

But we could also solve it as follows, which seems to be even easier and more straight forward now that i think of it :)

* Repository contains a valid version tag
  * Return the version of the tag
* else
  * fallback option set to a valid value
    * fallback is used
  * no fallback set
    * error
 
I'll edit the PR accordingly.